angular/headless/test.ts

```diff
@@ -1,10 +1,9 @@
-// This file is required by vitest
-
-import 'zone.js';
-import 'zone.js/testing';
+import '@angular/compiler';
```

Comment:
is this one required? usually it's for JIT

Reply:
Indeed this is required, as described here:
https://analogjs.org/fr/docs/features/testing/vitest#setup-for-running-tests-in-the-browser